composer.json

```diff
@@ -264,7 +268,8 @@
 		"wp-cli/wp-cli-bundle": "dev-master",
 		"wp-cli/wp-cli-tests": "dev-master",
 		"wp-cli/wp-config-transformer": "dev-master as 1.2.x-dev",
-		"wp-cli/wp-super-cache-cli": "dev-master"
+		"wp-cli/wp-super-cache-cli": "dev-master",
+		"wp-cli/maintenance-mode-command": "dev-master"
```

Comment:
Repos are alphabetically sorted  here. Can you push it upwards into the correct location?

Reply:
@schlessera Done.